examples/interface-to-external-code/interface_to_external_code.bal

```diff
@@ -0,0 +1,20 @@
+import ballerina/jballerina.java;
+
+// Returns the `out` field in the `java.lang.System` class.
+// `java.lang.System.out` is of the `java.io.printString` type.
```

Comment:
What is `java.io.printString`?

Reply:
It should be `PrintStream`. I will update.